components/brave_shields/content/browser/brave_shields_util.cc

```diff
@@ -908,7 +905,7 @@ mojom::FarblingLevel GetFarblingLevel(HostContentSettingsMap* map,
     case ControlType::BLOCK:
       return brave_shields::mojom::FarblingLevel::MAXIMUM;
     case ControlType::BLOCK_THIRD_PARTY:
-      NOTREACHED();
+      NOTREACHED_NORETURN();
```

Comment:
For the record, `NOTREACHED()` and `NOTREACHED_NORETURN()` are the same, but I guess may already know that.